sf-docker/LFIS/docker-compose.yml

```diff
@@ -97,6 +97,40 @@ services:
     volumes:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
 
+  sf-station:
```

Comment:
don't we want these changes also in jseton-LFIS?

Reply:
the Jetson does not have LFIS deployment pattern in the current GitHub

Reply:
wha is this then? https://github.com/innovatrics/smartface/tree/master/sf-docker/special/jetson-LFIS

Reply:
thank you for pointing that out. It is currently updated for Jetson LFIS as well.